fiftyone/utils/hrm2.py

```
@@ -0,0 +1,1947 @@
+    model = SMPL(
+        model_path=os.path.dirname(smpl_model_path),
+        gender="neutral",
+        batch_size=1,
+        create_transl=False,
+    )
+    logger.info(f"Loaded SMPL model from {smpl_model_path}")
+
+    return model
+
+
+def apply_hrm2_to_dataset_as_groups(
```

Pylint (4.0.3)

[refactor] Too many positional arguments (8/5)

(R0917)